rest-api/participant_enums.py

```diff
@@ -56,6 +56,53 @@ class SampleStatus(messages.Enum):
   """Status of biobank samples"""
   UNSET = 0
   RECEIVED = 1
+  # DA-814 - sample disposal statuses for a good outcome.
+  DISPOSED = 10
+  CONSUMED = 11
+  UNKNOWN = 12
+  # DA-814 - sample disposal statuses for a bad outcome.
+  SAMPLE_NOT_RECEIVED = 13
+  SAMPLE_NOT_PROCESSED = 14
+  ACCESSINGING_ERROR = 15
+  LAB_ACCIDENT = 16
+  QNS_FOR_PROCESSING = 17
+  QUALITY_ISSUE = 18
+
+def get_sample_status_enum_value(status):
+  """
+  Return the SampleStatus enum value for the given status from Mayo
+  :param status: a sample status value
+  :return: SampleStatus enum value
+  """
+  if status is None:
+    return SampleStatus.UNSET
+
+  # Set Received if we have an empty status value, the sample has not been disposed of yet.
+  if not status:
```

Comment:
The only time I could think this line would be True and the one above is not, is if status == False. Which it shouldn't be right ?

Reply:
`if not status` will also return true if status is an empty string.  In the biobank nightly file, the disposed status is empty if the sample is still active.